scripts/pr_redirect.html

```diff
@@ -0,0 +1,26 @@
+<!doctype html>
+<head><meta charset="utf-8">
+<title>Redirecting...</title>
+</head>
+<body>
+<h1>Redirecting...</h1>
+<p>
+  If you have JavaScript enabled, and you got here from a pull request to the tc39/ecma262 repository on GitHub, this page will attempt to redirect you to the appropriate snapshot rendering of the specification as of that pull request.
+</p>
+<p>
+  If it doesn't work, you can get there yourself by clicking the "Show all checks" at the bottom of the pull request, then clicking the "details" link next to the "netlify/ecma262-snapshots/deploy-preview" check, as in <a href="https://cdn.netlify.com/f0b93f3003d5669924b6e73540bbfe07bc3d8d95/34c5d/img/blog/deploy-preview-workflow.gif">this animation</a>.
+</p>
+<script>
+var match = /^https:\/\/github\.com\/tc39\/ecma262\/pull\/(\d+)/.exec(document.referrer);
```

Comment:
Unfortunately, Safari's ITP is gonna break this.

Reply:
What if we made a little script that edits the proper link into the pr description?

Reply:
[This blog post](https://webkit.org/blog/9521/intelligent-tracking-prevention-2-3/) says this only happens when "the referrer has link decoration", which I think shouldn't be a problem here (since github URLs do not tend to be decorated). Is that not the whole picture?

Reply:
@devsnek I'm not really a fan of editing other people's comments, even with a bot, but I guess that would be acceptable if this doesn't work.

Reply:
It’s more recent: https://webkit.org/blog/9661/preventing-tracking-prevention-tracking/

We could just have a bit that comments on the PR.

Reply:
I'm definitely not a fan of a comment bot.

If Safari's going to break this, this probably isn't a viable approach.

Reply:
Well that sucks.

Opened https://github.com/tc39/ecma262/issues/1883 for further discussion. I'm going to close this for now.